snap/info_snap_yaml_test.go

```diff
@@ -1336,21 +1336,65 @@ apps:
    post-stop-command: post-stop-cmd
    restart-condition: on-abnormal
    bus-name: busName
+   sockets:
+     sock1:
+       listen-stream: /tmp/sock1.socket
```

Comment:
Unfortunately the fact these files are absolute creates a more serious issue that we'll need to find a solution for. We cannot allow snaps to create arbitrary files in arbitrary locations, which is what this is now doing. Note that systemd is running out of any confinement, and will do whatever it's asked to do.

Reply:
What is worse those names will be created on the host and not in the execution environment. The /tmp/sock1.socket won't exist for the application at runtime.

Reply:
@zyga that's actually addressed in code, I forgot to fix those paths. Done now